### PR TITLE
FFC-206995: date review completed including testing date

### PR DESCRIPTION
**Pull request by marcin-mogiela**

### CODE REVIEW

**Review by govTechSatish — DISMISSED** [review]

File: app/routes/vet-visit-date.js [hunk 1]
```diff
@@ -42,12 +42,107 @@ module.exports = [{
...
+          })
+        }
+        return h
+          .view(templatePath, {
+            ...request.payload,
+            ...dateInputErrors,
+            errors,
+            whenCarriedOut: {
+              value: request.payload.whenCarriedOut,
+              errorMessage: error.details.find(e => e.context.label === 'whenCarriedOut')
```
Comment (liamwilson-kainos) [inline app/routes/vet-visit-date.js:122]: maybe overkill, but could maybe make the labels as string constants?

**Review by liamwilson-kainos — CHANGES_REQUESTED** [review]

File: app/routes/vet-visit-date.js [hunk 2]
```diff
@@ -61,14 +156,64 @@ module.exports = [{
...
+            },
+            month: {
+              value: request.payload['on-another-date-month']
+            },
+            year: {
+              value: request.payload['on-another-date-year']
+            }
+          }
+        }).code(400).takeover()
       }
```
[the remark this answers is not in the document]
Comment (marcin-mogiela) [reply app/routes/vet-visit-date.js:237]: done

File: app/views/vet-visit-date.njk [hunk 3]
```diff
@@ -15,25 +15,114 @@
 {% block content %}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
+      {% if errors.length %}
```
[the remark this answers is not in the document]
Comment (marcin-mogiela) [reply app/views/vet-visit-date.njk]: feature toggle added

File: package.json [hunk 4]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ffc-ahwr-farmer-claim",
-  "version": "0.33.2",
+  "version": "0.33.3",
```
Comment (chrissmKainos) [inline package.json:3]: This should probably be bumped up a minor version and not just patch.

**Review by liamwilson-kainos — DISMISSED** [review]

**Review by chrissmKainos — APPROVED** [review]

**Pull request merged**